src/System.IO.FileSystem.DriveInfo/tests/DriveInfo.Windows.Tests.cs

```diff
@@ -0,0 +1,194 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.IO;
+using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+using System.Runtime.InteropServices;
+using Xunit;
+using System.Text;
+
+namespace System.IO.FileSystem.DriveInfoTests
+{
+    public class DriveInfoTests
+    {
+        [Fact]
+        public void TestConstructor()
+        {
+            string[] invalidInput = { ":", "://", @":\", ":/", @":\\", "Az", "1", "a1", "\\share", "\\", "c ", string.Empty, " c" };
```

Comment:
should this be: \\\\share? I guess that you meant to test double backslash and not one escaped.